FirebaseMessaging/Sources/FIRMessaging.m

```diff
@@ -415,11 +415,9 @@ - (void)handleIncomingLinkIfNeededFromMessage:(NSDictionary *)message {
   SEL openURLWithOptionsSelector = @selector(application:openURL:options:);
   SEL openURLWithSourceApplicationSelector = @selector(application:
                                                            openURL:sourceApplication:annotation:);
-// TODO(Xcode 15): When Xcode 15 is the minimum supported Xcode version, it will be unnecessary to
-// check if `TARGET_OS_VISION` is defined.
-#if TARGET_OS_IOS && (!defined(TARGET_OS_VISION) || !TARGET_OS_VISION)
+#if TARGET_OS_IOS && !TARGET_OS_VISION
```

Comment:
re: _TARGET_OS_IOS is no longer 1 on visionOS_, would this shorten further to?

```suggestion
#if TARGET_OS_IOS
```

Reply:
You're right, thanks! Removed this one and the one below.